torch/csrc/api/include/torch/serialize.h

```diff
@@ -74,6 +74,7 @@ void save(const std::vector<torch::Tensor>& tensor_vec, SaveToArgs&&... args) {
 }
 
 TORCH_API std::vector<char> pickle_save(const torch::IValue& ivalue);
+TORCH_API IValue pickle_load(const std::function<bool(char*, size_t)>& reader);
```

Comment:
can this be implemented with a `using`? It is the same as torch::jit::pickle_load